tests/e2e/specs/stories-editor/featured-image-automatically-set.test.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * WordPress dependencies
+ */
+import { clickButton, createNewPost, getAllBlocks, selectBlockByClientId } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { activateExperience, deactivateExperience, insertBlock, uploadMedia } from '../../utils';
+
+const FEATURED_IMAGE_NOTICE_TEXT = 'Selecting a featured image is required.';
+const LARGE_IMAGE = 'large-image-36521.jpg';
+const MEDIA_LIBRARY_BUTTON = '.editor-media-placeholder__media-library-button';
+const SELECT_BUTTON = '.media-modal button.media-button-select';
+
+const getFeaturedImageFromDocument = () => page.evaluate( () => (
+	document.querySelector( '.editor-post-featured-image img' ).getAttribute( 'src' )
+) );
+
+const getFeaturedImageFromStore = () => page.evaluate( () => (
+	wp.data.select( 'core/editor' ).getEditedPostAttribute( 'featured_media' )
+) );
+
+/**
+ * Tests that the featured image is automatically set when selecting an image in certain places.
+ *
+ * These are similar to the tests in video-poster-image-extraction.test.js.
+ */
+describe( 'Featured Image Automatically Set', () => {
+	beforeAll( async () => {
+		await activateExperience( 'stories' );
+	} );
+
+	afterAll( async () => {
+		await deactivateExperience( 'stories' );
+	} );
+
+	describe( 'Image Block', () => {
+		it( 'on uploading a big enough image to the Core Image block, it should set it as the featured image', async () => {
```

Comment:
The `it` here at the beginning is meant to start the sentence:

```
it( 'should set it as the featured image when uploading a big enough to the image block' ...
```

Reply:
Oh, thanks for pointing that out.

Reply:
This is corrected in https://github.com/ampproject/amp-wp/pull/2927/commits/b9d00d11c16d6577d495ed5a97349d7c86165a20#diff-858da3b74103498c6a58eaa822f3c1d1L39